.travis.yml

```diff
@@ -109,6 +109,11 @@ deploy:
         branch: master
         condition: $BUILD_CONFIGURATION = Release && $(mono --version | perl -ne'/version (\d+\.\d+\.\d+)/ and print $1') = $BUILD_RELEASE_MONO_VERSION
 
+after_success:
+  - wget https://raw.githubusercontent.com/DiscordHooks/travis-ci-discord-webhook/master/send.sh
```

Comment:
Suggest moving the `wget` command inside the `if` so we don't download the script if we don't need it.

Reply:
It'll probably not need it when I refactor the lot to use stages and I _know_ this will work. Breaking it up across the lines didn't fit the few minutes I had spare to restore something I do keep an eye on.

Reply:
OK, I look forward to seeing the stages stuff.